subsys/sal/sid_pal/include/sid_mfg_hex_parsers.h

```diff
@@ -4,36 +4,40 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 #include <tlv/tlv.h>
+#include <sid_pal_mfg_store_ifc.h>
+
+#define MFG_FLAGS_TYPE_ID SID_PAL_MFG_STORE_VALUE_MAX - 1
+struct mfg_flags {
+	uint8_t UNUSED : 6;
+	uint8_t keys_in_psa : 1;
+	uint8_t initialized : 1;
+	uint8_t UNUSED_2[3];
```

Comment:
I think 6 unused bits on flags is engoud, we can always add a new tlv entry in the future

Reply:
I wanted to keep this as 4 bytes, as it is minimal size for entry in tlv, but I can also reduce it ( the other 3 bytes will just became padding

Reply:
ok, if it's another module limitarion let it stay.
Just please use lowercase name, possibly without `_2` suffix